src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/domain/camunda/response/CancellationEvaluateResponse.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.wacaseeventhandler.domain.camunda.response;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.ToString;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.camunda.DmnValue;
+
+@ToString
+@Builder
+@SuppressWarnings("PMD.UseConcurrentHashMap")
```

Comment:
a small one, but do you need to suppress this here, it does not seem you use any Map at all